src/test/java/com/wikia/webdriver/common/core/Assertion.java

```diff
@@ -34,6 +34,12 @@ public static boolean assertStringContains(String current, String pattern) {
     return assertion;
   }
 
+  public static void assertEqualsIgnoreCase(String current, String pattern) {
+    current = current.toLowerCase();
+    pattern = pattern.toLowerCase();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "pattern". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)